docs/README.md

```diff
@@ -0,0 +1,5 @@
+# Community Docs
```

Comment:
I think the discord link should be added here or to the main README.md.

Reply:
Sure, why not.